packages/@aws-cdk/cloudformation-include/test/test-templates/fn-sub-parameter-shadow.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Call this file `fn-sub-shadow-parameter.json`, to bring it closer to existing test files.